readthedocs/search/faceted_search.py

```diff
@@ -21,7 +16,11 @@
 
 class RTDFacetedSearch(FacetedSearch):
 
-    def __init__(self, user, **kwargs):
+    """Custom wrapper around FacetedSearch."""
+
+    operators = []
+
+    def __init__(self, query=None, filters=None, user=None, **kwargs):
```

Comment:
also, user and filter_by_user isn't being used in neither, .org or .com. I'll remove that after this PR is merged.